### ISSUE TITLE
Double tap to zoom in

### ISSUE THREAD

**Issue by thany** [mapsme/omim#7570]
Would be nice if a simple double tap would zoom in. I think people kind of expect this from a map.

Can it be added?

**Comment by rokuz** [comment]
Hi! Already implemented, isn't work on your device?

**Comment by thany** [comment]
Nope. It so.etimea zooms in or out a little bit. But that's totally random.

Double tap & drag up/down works fine, and I suspect it tries going into that mode, discarding the double tap gesture.

**Comment by rokuz** [comment]
Just to clear, do you do it this way?
https://youtu.be/x-qTb21AHlo

**Comment by thany** [comment]
Yup, exactly like that. Except I'm on a Galaxy S7.

**Comment by jaller94** [comment]
There are two actions:
1. Zooming in (animated) by double tapping on the same position on the display and
2. Zooming in/out (jump) by double tapping where the 2nd position is further up (zoom out) or further down (zoom in).

The positions need to be both vertically and horizontally close to each other to be recognized as the first action.
In a quick test I did not find any difference between the required precision when changing Android's setting "Display size". Both actions work for me, however, I do trigger the second on on accident every fifth double tap.

**Comment by thany** [comment]
Two months later. Any update on this?
